### PR TITLE
Support squeezed label as input in paddle.metric.Accuracy

### PR DESCRIPTION
**Pull request by qingqing01**
### PR types
Bug fixes

### PR changes

APIs

### Describe

- Support squeezed label in paddle.metric.Accuracy
- Add unit testing
- Update doc in cifar10

### CODE REVIEW

File: python/paddle/vision/datasets/cifar.py [hunk 1]
```diff
@@ -53,7 +53,9 @@ class Cifar10(Dataset):
             :attr:`data_file` is not set. Default True
 
     Returns:
-        Dataset: instance of cifar-10 dataset
+        Dataset: instance of cifar-10 dataset. If transform is None, the shape
+            of each data iterm is [3, 32, 32], and default dtype is float32.
```
Comment (LielinJiang) [inline python/paddle/vision/datasets/cifar.py]: 默认是返回pil的图片，shape应该是[32, 32, 3]

**Review by heavengate — DISMISSED** [review]

File: python/paddle/vision/datasets/cifar.py (hunk 1 above)
Comment (qingqing01) [reply python/paddle/vision/datasets/cifar.py]: Code is:


图片

Comment (LielinJiang) [reply python/paddle/vision/datasets/cifar.py]: https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/vision/datasets/cifar.py#L158-L171，是不是代码没更新

Comment (qingqing01) [reply python/paddle/vision/datasets/cifar.py]: @LielinJiang  Done. Thanks!

**Review by heavengate — DISMISSED** [review]

**Review by LielinJiang — DISMISSED** [review]

**Review by saxon-zh — DISMISSED** [review]
LGTM

**Review by LielinJiang — APPROVED** [review]

**Review by saxon-zh — APPROVED** [review]
LGTM

**Pull request merged**